qa/zcash/full-test-suite.sh

```diff
@@ -39,6 +39,9 @@ run_test_phase make check '||' \
                false ';' \
                '}'
 
+run_test_phase "${REPOROOT}/src/zerocash/tests/zerocashTest"
+run_test_phase "${REPOROOT}/src/zerocash/tests/test_zerocash_pour_ppzksnark"
+
```

Comment:
ACK. I assume we're also removing their buildbot builder since they'll now be in our ``full-test-suite``?